### PR TITLE
OKTA-620570 : fix : Enable button text wrapping on consent pages

### PR DESCRIPTION
**Pull request by glenfannin-okta**
## Description:
The purpose of this PR is to fix an issue with the button text being hidden in alternate languages because it extends past the fixed width of the button.

Before:
image

After fix (displays english to show no impact and alternate languages to show the fix):

https://github.com/okta/okta-signin-widget/assets/97472729/da012995-bb79-42f2-a404-152254fbb5a3

After speaking with @alexdahl-okta it was suggested to correct the line-height for the button text, here are the screenshots of the updated view: 

Screen Shot 2023-07-27 at 6 08 41 PM
Screen Shot 2023-07-27 at 6 09 54 PM
Screen Shot 2023-07-27 at 6 10 01 PM
Screen Shot 2023-07-27 at 6 10 07 PM

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-620570](https://oktainc.atlassian.net/browse/OKTA-620570)

### Reviewers:
cc: @alexdahl-okta

### CODE REVIEW

File: assets/sass/modules/_admin-consent.scss [hunk 1]
```diff
@@ -99,6 +99,9 @@ $image-size: 30px;
 
     input {
       width: 140px;
+      height: auto;
+      min-height: 50px;
```
Comment (leemchale-okta) [inline assets/sass/modules/_admin-consent.scss:103]: is `50px` the default height of the button?

Comment (glenfannin-okta) [reply assets/sass/modules/_admin-consent.scss:103]: That value was derived from the _btns.scss file, that was the explicit set height of the button so to not modify the size, I reset it here as the min-height to ensure it stays the same when the text does not need to wrap.

Comment (leemchale-okta) [reply assets/sass/modules/_admin-consent.scss:103]: awesome sounds good

**Review by leemchale-okta — APPROVED** [review]

**Comment by shawyu-okta** [comment]
@glenfannin-okta is this an issue on g3?

**Comment by glenfannin-okta** [comment]
> @glenfannin-okta is this an issue on g3?

No sir, and mostly because the buttons in G3 are stacked, but even when the widget is reduced down in size the text still wraps as expected because of MUI/Ody

Reduced size:
image

Normal: 
image

**Comment by shawyu-okta** [comment]
> No sir, and mostly because the buttons in G3 are stacked, but even when the widget is reduced down in size the text still wraps as expected because of MUI/Ody


@glenfannin-okta 🎉

**Pull request merged**